content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+
+
+
+**Properties:**
+
+
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `additional_settings`
+    Optional additional settings. Possible fields in dictionary are: port, request_path, timeout_sec, check_interval_sec, healthy_threshold, unhealthy_threshold.
+
+    *default:* {}
+  * `health_check_type`
+    This field indicates if this health check is a HTTP or HTTPS based health check. Possible values are: 'http' and 'https'.
+
+    *type:* string
+    *default:* http
+  * `use_external_resource`
+    Indicate whether the resource exists and use existing (true) or if Cloudify should create new resource (false).
+
+    *type:* boolean
+    *default:* False
+  * `name`
+    Optional health check name. By default it will be health check id.
+
+    *type:* string
+    *default:*
+
+
+
+
+
+## cloudify.gcp.nodes.Image
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+A stored image which can be used as the base for newly created Instances.
+
+
+
+**Properties:**
+
+
+  * `image_name`
+    Name to use for the image. Defaults to the instance ID.
+
+    *default:*
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `additional_settings`
+    Additional setting for image
+
+    *default:* {}
+  * `image_path`
+    The (local system) path to the image file which will be uploaded.
+
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists or if Cloudify should create the resource.
+
+    *type:* boolean
+    *default:* False
+
+
+
+
+
+## cloudify.gcp.nodes.Instance
+**Derived From:** [cloudify.nodes.Compute]({{< relref "blueprints/built-in-types.md" >}})
+
+A GCP Instance (i.e. a VM).
+
+
+**Properties:**
+
+
+  * `scopes`
+    Optional scopes. If not will set by default:  'https://www.googleapis.com/auth/devstorage.read_write', 'https://www.googleapis.com/auth/logging.write'
+
+    *default:* []
+  * `instance_type`
+    The instance's type. All available instance types can be found here:  https://cloud.google.com/compute/docs/machine-types
+
+    *type:* string
+    *default:* n1-standard-1
+  * `name`
+    Optional instance name. By default it will be instance id.
+
+    *type:* string
+    *default:*
+  * `zone`
+    Optional zone name. If not given, this instance will be deployed in default zone.
+
+    *type:* string
+    *default:*
+  * `tags`
+    Optional tags. If not given, this instance will have a tag only with its name.
+
+    *type:* string
+    *default:*
+  * `external_ip`
+    Should the Instance be created with an externally-accessible IP address. This will be an ephemeral IP. If you would like to use an IP address which can be transferred to another Instance then connect this Instance to an `Address` node using the `cloudify.gcp.relationships.instance_connected_to_ip` relationship.
```

Comment:
~~Should~~ **Specifies whether** the Instance **is to** be created with an externally-accessible IP address. This will be an ephemeral IP. ~~If you would like to~~ **To** use an IP address ~~which~~ **that** can be transferred to another Instance **,** ~~then~~ connect this Instance to an `Address` node using the `cloudify.gcp.relationships.instance_connected_to_ip` relationship.